### PR TITLE
Ai photogram ajax

### PR DESCRIPTION
**Pull request by aizatibraimova**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -59,23 +62,27 @@ def destroy
     respond_to do |format|
       format.html { redirect_back fallback_location: root_url, notice: "Comment was successfully destroyed." }
       format.json { head :no_content }
+      format.js do
```
Comment (ahuynh3a) [inline app/controllers/comments_controller.rb:65]: good job being mindful of white spaces and removing them when unnecessary!

File: app/views/comments/create.js.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+var added_comment = $("<%= j(render @comment) %>");
```
Comment (ahuynh3a) [inline app/views/comments/create.js.erb:1]: good job utilizing variables! It makes your code easy to read!

File: app/views/comments/edit.js.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+$("#<%= dom_id(@comment) %>").replaceWith("<%= j(render "comments/form", comment: @comment) %>");
```
Comment (ahuynh3a) [inline app/views/comments/edit.js.erb:1]: Great job here! You can assign your jquery to variables here to make your code clearer to read!

File: app/views/follow_requests/_form.html.erb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<%= form_with(model: follow_request, class: "d-inline-block") do |form| %>
+<%= form_with(model: follow_request, local: false, class: "d-inline-block") do |form| %>
```
Comment (ahuynh3a) [inline app/views/follow_requests/_form.html.erb:1]: good job being mindful of when to use local: false and remote:true

**Review by ahuynh3a — APPROVED** [review]
great job Aizat! You code is clean and easy to follow!

File: app/controllers/follow_requests_controller.rb [hunk 5]
```diff
@@ -54,6 +56,7 @@ def destroy
     respond_to do |format|
       format.html { redirect_back fallback_location: root_url, notice: "Follow request was successfully destroyed." }
       format.json { head :no_content }
+      format.js
```
Comment (aborov) [inline app/controllers/follow_requests_controller.rb:59]: Nice work setting up your actions in controllers to respond with JS

File: app/views/comments/_comment.html.erb [hunk 6]
```diff
@@ -1,21 +1,22 @@
...
       <% if current_user == comment.author %>
-        <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
+        <%= link_to edit_comment_path(comment), 
+         class: "btn btn-link btn-sm text-muted", 
+         remote: true do %>
           <i class="fas fa-edit fa-fw"></i>
         <% end %>
-
-        <%= link_to comment, data: { turbo_method: :delete }, class: "btn btn-link btn-sm text-muted" do %>
+        <%= link_to comment, method: :delete, class: "btn btn-link btn-sm text-muted", remote: true do %>
```
Comment (aborov) [inline app/views/comments/_comment.html.erb:19]: Great job breaking the link with `remote: true`

File: app/views/comments/create.js.erb [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+added_comment.hide();
+
+$("#<%= dom_id(@comment.photo) %>_new_comment_form").before(added_comment);
+
+added_comment.slideDown();
+
+$("#<%= dom_id(@comment.photo) %>_new_comment_form #comment_body").val("");
```
Comment (aborov) [inline app/views/comments/create.js.erb:9]: Wow, this does make it easier to read. Maybe I should do it too!

File: app/views/follow_requests/create.js.erb [hunk 8]
```diff
@@ -0,0 +1,4 @@
+
+var current_state = $(".<%= dom_id(@follow_request.sender) %>_<%= dom_id(@follow_request.recipient) %>_follow_unfollow");
+var updated_state = $("<%= j(render "follow_requests/follow_unfollow", sender: @follow_request.sender, recipient: @follow_request.recipient) %>");
+current_state.replaceWith(updated_state);
```
Comment (aborov) [inline app/views/follow_requests/create.js.erb:4]: That's a nice way of assigning variables to keep it clear on what's going on!

**Review by aborov — APPROVED** [review]
Great job Aizat. Reading your code is a pleasure!

**Pull request merged**